<?xml version="1.0" ?>
<launch>
    <arg name="robot_ip" />
    <arg name="load_gripper" default="false" />
    <include file="$(find franka_control)/launch/franka_control.launch">
        <arg name="robot_ip" value="$(arg robot_ip)" />
        <arg name="load_gripper" value="$(arg load_gripper)" />
    </include>

    <node name = "netft_node" pkg = "netft_control" type = "netft_node"
        args = "--address 192.168.1.1 --rate 100" output = "screen">
    </node>

    <rosparam command="load" file="$(find franka_example_controllers)/config/franka_example_controllers.yaml" />
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="variable_impedance_controller"/>
    <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" required="false" />
</launch>